### DISCUSSION TITLE
Kong redirects instead of reverse proxy

### DISCUSSION THREAD

**Discussion by anirudh-ta15** [Kong/kong#8309] [Help] [answered]
Hi team,

We have setup kong to reverse proxy apache tomcat. 
When request is hit, rather than reverse proxying, it redirects to tomcat destination url I.e url is changing from example.com -> web.com rather than staying on example.com

Kong cluster runs on example.com
Tomcat runs on web.com

Do we need to add any special configuration on Kong or tomcat to reverse proxy?

**Comment by hbagdi** [comment, marked as answer]
Take a look at `preserve_host` property on the Route resource.
